pkg/logs/internal/launchers/file/launcher.go

```diff
@@ -342,10 +342,10 @@ func (s *Launcher) createRotatedTailer(t *tailer.Tailer, file *tailer.File, patt
 }
 
 func CheckProcessTelemetry(stats *util.ProcessFileStats) {
-	if stats.AgentOpenFiles/stats.OsFileLimit > 0.9 {
-		log.Warnf("Agent process is close to OS file limit of %v. Agent process currently has %v files open.", stats.OsFileLimit, stats.AgentOpenFiles)
-	} else if stats.AgentOpenFiles/stats.OsFileLimit >= 1 {
-		log.Errorf("Agent process has reached the OS open file limit: %v. This may be preventing log files from being tailed by the Agent and could interfere with the basic functionality of the Agent. OS file limit must be increased.", stats.OsFileLimit)
+	if stats.AgentOpenFiles/stats.OsFileLimit > 0.7 {
+		log.Warnf("Agent process is over 70%% of OS open file limit (%v). Agent process currently has %v files open. Consider increasing OS file limit.", stats.OsFileLimit, stats.AgentOpenFiles)
```

Comment:
lets print out the current percentage in this log in addition to the raw numbers. 

Since we start printing at 70%, printing the percentage could help estimate how quickly the agent is reaching the limit, rather than having to do that math manually.